omnibus/.kitchen.yml

```diff
@@ -9,7 +9,8 @@ driver:
 
 provisioner:
   name: chef_zero
-  require_chef_omnibus: 13.6.4
+  product_name: chef
+  product_version: 13.6.4
```

Comment:
We should probably update this to 14 since that's what's being used.

Reply:
Hmm. I am currently confused as to how Chef 14 is getting into the omnibus package. 

```
~$ sudo supermarket-ctl reconfigure
Starting Chef Client, version 14.3.37
resolving cookbooks for run list: ["omnibus-supermarket::default"]
...
```

And yet ...

```
# /opt/supermarket/version-manifest.txt
supermarket 3.1.74+20180815175347

Component                Installed Version       Version GUID                                                              Overridden From
---------------------------------------------------------------------------------------------------------------------------------------------
...
cacerts                  2018-01-17              sha256:defe310a0184a12e4b1b3d147f1d77395dd7a09e3428373d019bef5d542ceba3
chef-gem                 13.6.4                  11.12.2
config_guess             master                  git:84f04b02a7e2fc8eaa9d52deee5f6d57b06fe447
...
```

Reply:
Isn't the kitchen file just for testing?

Reply:
It is. This one is a kitchen for spinning up local omnibus builders and then performing the omnibus build. The Chef version in this one probably doesn't matter and could move to 14.

Reply:
So ..uh.. yea, we can move it up. The version of Chef inside the omnibus package is a separate issue.